src/lnp/resources/test.template.properties

```diff
@@ -10,6 +10,17 @@
 # Note: the '.' in the url has been replaced by -dot- to support https connection for the staging server.
 test.app.url=http\://localhost\:8080
 
+# This is the domain of the app we are testing against.
+# e.g. test.app.domain=localhost
+# e.g. test.app.domain=7-0-0-dot-teammates-john.appspot.com
+test.app.domain=localhost
```

Comment:
not sure if it is necessary to add this, given that you can parse from line 11 above.

Reply:
I think doing this is much simpler than parsing since the staging server's url does not have the port attached to it like the local url

Reply:
What is simpler from the user (dev who use LnP script) perspective is that he only needs to update 1 url.

Reply:
Sorry, but @moziliar is right. This solution is not the optimal one.

There are utility functions used in other test places to check whether the URL indicates a staging server. With that, it's simple to decide whether to pass the port as `null` or `8080`. Notice that there is no need to set the port as dynamic, because unless the person modifies `build.gradle`, the port is guaranteed to be `8080`.

Reply:
@daongochieu2810 @moziliar Since the PR has been merged, one of you could help to open up an issue targeted at parsing the necessary information for the LnP tests from this PR

Reply:
Ok, will do then